drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -242,6 +242,11 @@ static inline int stm32_clock_control_configure(const struct device *dev,
 		return err;
 	}
 
+	if (pclken->enr == NO_SEL) {
```

Comment:
What is a usecase where pclken->enr is NO_SEL is it a "default value"

Reply:
See commit description:
 USB clock on PLL_Q output on F405 devices is not selectable. But yet it needs to be described in device tree in order to be able to call `clock_control_get_rate`.